### PR TITLE
133 client qa updates

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Adds new `Checked on Frontend?` field to CL Mailchimp ACF JSON that defaults to having newsletter be checked on the frontend https://github.com/INN/umbrella-citylimits/commit/28e13093c4a0b14b626774f0613f99ea6dd9f3f5
- Hides the hero image on author archive template https://github.com/INN/umbrella-citylimits/commit/c892f10398f80f4c742c201820c88a97ff67296c
- Updates `.and` to be lowercased https://github.com/INN/umbrella-citylimits/commit/c892f10398f80f4c742c201820c88a97ff67296c
- Reduces size of the top featured story title on the homepage https://github.com/INN/umbrella-citylimits/commit/7ce574c5e6cc9e2a9f81ba011afb7409f93d2762
- Copies `baseFontSize: 16px` from Largo in child theme `variables.less` and uses it to set the byline font size on the special projects seven posts widget so it's the same as on archive pages https://github.com/INN/umbrella-citylimits/commit/77e736044aa0dc52bb51c7183df8f30471ee5336
- https://github.com/INN/umbrella-citylimits/commit/74e55a1a24447d20efd072889f1533128d31f601 updates the `Share via email` mailto: link to:
```
Email Subject is: CityLimits.org: [HEADLINE]

Email Body is:
From City Limits:
[HEADLINE]
[DEK]
[Link]
``` 

## Why

For #133

## Testing/Questions

Features that this PR affects:

- Newsletter signup form
- Author archive template
- Bylines
- Homepage top featured story
- Share via email button

Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository?

Steps to test this PR:

**Mapping the Future should be checked by default in the Newsletter signup:**
1. Make sure the new `Checked on Frontend?` field is appearing in the CL Mailchimp editor. If not, you'll need to add it in via the Custom Fields editor. Settings for it can be found here: https://github.com/INN/umbrella-citylimits/commit/28e13093c4a0b14b626774f0613f99ea6dd9f3f5
2. Make sure the checkboxes toggle checked/unchecked correctly based on their `Checked on Frontend?` field

**Remove lede image from author landing page template:**
1. Make sure the hero image is hidden on author archive pages

**Change "And" to "and" on article page main bylines:**
1. Find a post with multiple authors and verify the `and` is lowercased

**Reduce size of top headline on mobile homepage so its more in line with headlines for subsequent stories:**
1. View the homepage on mobile and verify the top featured story looks ok

**Update byline size on series landing pages so that it matches the sizes on category landing pages:**
1. Add the `City Limits Seven Series Posts` widget to a series page and make sure bylines are checked to be shown
2. Verify the byline font size is the same as the byline font size on category archives

**Change email share to use this format:**
1. Go to a post
2. Click the email share button
3. Verify all of the content is correct and formatting looks ok

### CODE REVIEW

File: wp-content/themes/citylimits/inc/post-social.php [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+            rawurlencode( html_entity_decode( get_the_title(), ENT_QUOTES, get_option('blog_charset') ) ),
+            rawurlencode( get_permalink() ),
+            $via,
+            __( 'Tweet', 'largo' )
+        );
+    }
+
+    if ( isset( $utilities['email'] ) && '1' === $utilities['email'] ) {
+        $output .= sprintf(
+            '<span data-service="email" class="email share-button"><a href="mailto:?subject=%2$s%3$s&body=%4$s%3$s%0D%0A%5$s%20%0D%0A%6$s" target="_blank"><i class="icon-mail"></i> <span class="hidden-phone">%1$s</span></a></span>',
```
Comment (benlk) [inline wp-content/themes/citylimits/inc/post-social.php]: What I get from this in the post body is:

> From City Limits: Video: Brooklyn’s Role in the ’18 Vote & What the Results Mean for the BoroKings County was center stage on Election Night 2018, helping to flip the State Senate and the U.S. House blue. What drove the electoral shift behind those results, and what will their impact be on the borough? We asked two veteran journalists.
> https://citylimits.test/2018/11/08/video-brooklyns-role-in-the-18-vote-what-the-results-mean-for-the-boro/

There's no spacing between the headline and the excerpt.

The body string there is `%4$s%3$s%0D%0A%5$s%20%0D%0A%6$s`:

- %4$s - `'From City Limits '`
- %3$s - post title
- %0D%0A - carriage return, line feed
- %5$s - excerpt
- %20%0D%0A - space, carriage return, line feed
- %6$s - permalink

Adding a `%20` before the first `%0D%0A` solves the lack of space. I'm not sure why it's needed, though.

**Comment by benlk** [comment]
> Mapping the Future should be checked by default in the Newsletter signup:
> 
> 1. Make sure the new Checked on Frontend? field is appearing in the CL Mailchimp editor. If not, you'll need to add it in via the Custom Fields editor. Settings for it can be found here: 28e1309
> 2. Make sure the checkboxes toggle checked/unchecked correctly based on their Checked on Frontend? field

The field wasn't automatically imported upon checkout; is there a way to trigger the import of the field?

> Hides the hero image on author archive template

Where was it shown before?

**Comment by joshdarby** [comment]
> The field wasn't automatically imported upon checkout; is there a way to trigger the import of the field?

@benlk I'm not sure 😕 I would say it's probably easier to just add the field manually on deploys to staging and prod

> re: header: Where was it shown before?

Screen Shot 2019-12-16 at 4 32 10 PM

It was shown above the author name.

Here's the markup for it
```
<div class="hero is-image ">
	<img width="600" height="400" src="https://citylimits.test/wp-content/uploads/2014/10/charter.jpg" class="attachment-full size-full wp-post-image" alt="" srcset="https://citylimits.test/wp-content/uploads/2014/10/charter.jpg 600w, https://citylimits.test/wp-content/uploads/2014/10/charter-336x224.jpg 336w" sizes="(max-width: 600px) 100vw, 600px">				<p class="wp-caption-text">Opportunity Charter School, with 44 percent teacher attrition in 2011-12, actually saw lower turnover than 18 other city charter schools—at least according to state data, which some schools dispute.</p>
		</div>
```

File: wp-content/themes/citylimits/partials/newsletter-signup-maincolumn.php [hunk 2]
```diff
@@ -22,15 +22,14 @@
...
 						foreach ( get_field('newsletter_group', 'option') as $group ) {
 							foreach ( $group['newsletters'] as $newsletter) {
 								if (!$newsletter['active']) {
 									continue;
 								}
 								$checked = '';
-								if (!$checkedFlag) {
+								if ( $newsletter['checked_on_frontend'] ) {
 									$checked = ' checked="checked"';
 									$checkedFlag = true;
```
[the remark this answers is not in the document]
Comment (joshdarby) [reply wp-content/themes/citylimits/partials/newsletter-signup-maincolumn.php]: Whoops. Nope. Removed in https://github.com/INN/umbrella-citylimits/pull/135/commits/1b6e32064a9c7e1c8b869934fd3b651cc7cb582b

File: wp-content/themes/citylimits/inc/post-social.php (hunk 1 above)
Comment (joshdarby) [reply wp-content/themes/citylimits/inc/post-social.php]: Good catch. https://github.com/INN/umbrella-citylimits/pull/135/commits/95e00a6dd865e096c312003ed78aabe9f0ae28dc adds in an extra space there, and also adds a comment with your notes about the order of that string for whoever has to modify it next 😅 

Why that extra space is required... I'd think it probably has to do with what app/email provider is rendering the body string. For example, `%0D%0A` is supposed to be a return line but Gmail (browser, at least) ignores it.

**Comment by benlk** [comment]
For Helen Zelon, is the author archive template for some reason displaying the featured image from the first post in The Loop? https://citylimits.org/author/helen-zelon/

The CSS fix here works, but I'm not sure why we're seeing that image there in the first place.

**Comment by joshdarby** [comment]
@benlk It looks like it might be a bug on the Largo `archive.php` template

https://github.com/INN/largo/blob/f50df16e9e06fb9402802e2f1ce76ce058d26d76/archive.php#L78-L79

`$post_id` comes back as `NULL` for `largo_get_term_meta_post` there so it probably is defaulting to the first post featured image in the loop

**Comment by benlk** [comment]
Thanks for identifying the source of that bug, Josh! https://github.com/INN/largo/issues/1838

**Pull request merged**